moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js

```diff
@@ -275,309 +277,537 @@ const ProjectsListViewTable = ({ query, searchTerm }) => {
+  //         <span key={tag} style={{ display: "block" }}>
+  //           {tag}
+  //         </span>
+  //       ));
+  //     },
+  //   },
+  //   {
+  //     title: "Created by",
+  //     field: "added_by",
+  //     hidden: hiddenColumns["added_by"],
+  //     emptyValue: "-",
+  //   },
+  //   {
+  //     title: "Public process status",
+  //     field: "public_process_status",
+  //     hidden: hiddenColumns["public_process_status"],
+  //     emptyValue: "-",
+  //   },
+  // ];
+
+  /**
+   * Handles the header click for sorting asc/desc.
+   * @param {int} columnId
+   * @param {string} newOrderDirection
+   * Note: this is a GridTable function that we are using to override a Material Table sorting function
+   * Their function call uses two variables, columnId and newOrderDirection. We only need the columnId
+   **/
+  // const handleTableHeaderClick = (columnId, newOrderDirection) => {
+  //   // Before anything, let's clear all current conditions
+  //   query.clearOrderBy();
+  //   const columnName = columns[columnId]?.field;
+
+  //   // Resets pagination to 0 when user clicks a header to display most relevant results
+  //   setPagination({
+  //     limit: query.limit,
+  //     offset: query.offset,
+  //     page: 0,
+  //   });
+
+  //   if (sort.column === columnName) {
+  //     // If the current sortColumn is the same as the new
+  //     // then invert values and repeat sort on column
+  //     setSort({
+  //       order: sort.order === "desc" ? "asc" : "desc",
+  //       column: columnName,
+  //       columnId: columnId,
+  //     });
+  //   } else if (sort.column !== columnName) {
+  //     // Sort different column in same order as previous column
+  //     setSort({
+  //       order: sort.order,
+  //       column: columnName,
+  //       columnId: columnId,
+  //     });
+  //   }
+  // };
+
+  /*
+   * Store column configution before data change triggers page refresh
+   * or opening advanced search dropdown triggers page refresh
+   */
+  useEffect(() => {
+    const storedConfig = JSON.parse(localStorage.getItem("mopedColumnConfig"));
+    if (storedConfig) {
+      setHiddenColumns(storedConfig);
+    }
+  }, [data, advancedSearchAnchor]);
+
+  const dataGridColumns = [
     {
-      title: "ID",
+      headerName: "ID",
       field: "project_id",
-      hidden: hiddenColumns["project_id"],
     },
     {
-      title: "Name",
+      headerName: "Name",
       field: "project_name",
-      hidden: hiddenColumns["project_name"],
-      render: (entry) => (
+      renderCell: (entry) => (
         <RouterLink
-          to={`/moped/projects/${entry.project_id}`}
+          to={`/moped/projects/${entry.row.project_id}`}
           state={{
             filters: Object.keys(filters).length
               ? btoa(JSON.stringify(filters))
               : false,
           }}
           className={classes.colorPrimary}
         >
-          {entry.project_name}
+          {entry.value}
```

Comment:
entry.value is whatever the field is, but i could have also entry.row.project_name here.